docs/source/configuration.rst

```diff
@@ -707,15 +712,13 @@ the flag :attr:`~CoreSection.log_raw` to true::
     log_raw = on
 
 In that case, IRC messages received and sent are stored into a file named
-``<base>.raw.log``, located in the log directory.
-
-The ``<base>`` prefix refers to the configuration's
-:attr:`~sopel.config.Config.basename` attribute.
```

Comment:
Hm. That information would be interesting to keep somewhere: knowing that a plugin can reuse `sopel.config.Config.basename` would be useful, I think.

Reply:
See latest (force-)push. Tried to balance the information delivery by moving this to a `.. note::` at the end, so it's out of the way when normal users are just trying to learn about settings.